examples/async-custom-entry/src/wasm.rs

```diff
@@ -16,6 +16,20 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+//! This program demonstrates the use of async `init` entry point, as well as custom entry point
```

Comment:
I would add some information about the execution. Docs should give some more description on async execution flow in this example.